dags/common/sftp_service.py

```diff
@@ -55,9 +57,22 @@ def __exit__(self, exc_type, exc_value, tb):
             return False
         return True
 
-    def list_files(self):
+    def store_files_name(self, fname):
+        self.file_names.append(re.sub(self.dir + "/", "", fname))
+
+    def list_files(self, excluded_dirs=[]):
```

Comment:
Not None because excluded_dirs in default params is []

Reply:
We have discussed this in the past, it's not a good practise to have `mutable` data structures in the signatures